### PR TITLE
Set YARN_IGNORE_PATH=1

### PR DESCRIPTION
**Pull request by christianscott**

### CODE REVIEW

File: internal/node/node_repositories.bzl [hunk 1]
```diff
@@ -591,11 +591,7 @@ if %errorlevel% neq 0 exit /b %errorlevel%
 
     # The entry points for yarn for osx/linux and windows.
     # Runs yarn using appropriate node entry point.
-    # Unset YARN_IGNORE_PATH before calling yarn incase it is set so that
-    # .yarnrc yarn-path is followed if set. This is for the case when calling
-    # bazel from yarn with `yarn bazel ...` and yarn follows yarn-path in
-    # .yarnrc it will set YARN_IGNORE_PATH=1 which will prevent the bazel
-    # call into yarn from also following the yarn-path as desired.
+    # Set YARN_IGNORE_PATH=1 so it doesn't go throug our .yarn/releases/yarn.sh wrapper
```
Comment (Silic0nS0ldier) [inline internal/node/node_repositories.bzl:594]: nit: `throug` -> `through`
Should also generalise comment.

File: internal/npm_install/npm_install.bzl [hunk 2]
```diff
@@ -743,17 +743,13 @@ def _yarn_install_impl(repository_ctx):
     # The entry points for npm install for osx/linux and windows
     if not is_windows_host:
         # Prefix filenames with _ so they don't conflict with the npm packages.
-        # Unset YARN_IGNORE_PATH before calling yarn incase it is set so that
-        # .yarnrc yarn-path is followed if set. This is for the case when calling
-        # bazel from yarn with `yarn bazel ...` and yarn follows yarn-path in
-        # .yarnrc it will set YARN_IGNORE_PATH=1 which will prevent the bazel
-        # call into yarn from also following the yarn-path as desired.
+        # Set YARN_IGNORE_PATH=1 so it doesn't go throug our .yarn/releases/yarn.sh wrapper
```
Comment (Silic0nS0ldier) [inline internal/npm_install/npm_install.bzl:746]: Same as before
> nit: `throug` -> `through`
> Should also generalise comment.

**Review by Silic0nS0ldier — APPROVED** [review]

File: internal/node/node_repositories.bzl (hunk 1 above)
Comment (christianscott) [reply internal/node/node_repositories.bzl:594]:
```suggestion
    # Set YARN_IGNORE_PATH=1 so it doesn't go through wrappers set by `yarn-path` https://classic.yarnpkg.com/lang/en/docs/yarnrc/#toc-yarn-path
```

File: internal/npm_install/npm_install.bzl (hunk 2 above)
Comment (christianscott) [reply internal/npm_install/npm_install.bzl:746]:
```suggestion
        # Set YARN_IGNORE_PATH=1 so it doesn't go through wrappers set by `yarn-path` https://classic.yarnpkg.com/lang/en/docs/yarnrc/#toc-yarn-path
```

**Pull request merged**